x-pack/test/apm_api_integration/basic/tests/observability_overview/has_data.ts

```diff
@@ -10,6 +10,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
 
+  const archiveName = 'apm_8.0.0';
```

Comment:
As mentioned I think we should pin this to a specific version. 
The current change is a good example: archive 8.0.0 is being replaced by archive 8.0.0. It's impossible to see what changed. At a minimum we should have a date that accompanies the archive if we choose a non-released, floating version.

Reply:
Btw. why is the archive called `apm_8.0.0`? Is the apm prefix needed to distinguish from other solutions?

Reply:
This is actually not an archive update, but simply using our main archive. Fwiw, if we have a more specific version in our name, it will be harder for us to automate an archive update.

Reply:
I don't think we need it, I think my intention was to distinguish from other archives like the observability one.

Reply:
okay i see it's not an archive update (they both already exist) but how do we distinguish between the two?

<img src="https://user-images.githubusercontent.com/209966/93202523-1d0da080-f753-11ea-9886-16cab36c5da9.png" width="300">

I assume we want to delete the old one

> Fwiw, if we have a more specific version in our name, it will be harder for us to automate an archive update.

Would this also be the case if we add a date to it?

Reply:
@sqren I would like to delete 8.0.0, but it's used in conjunction with rum_8.0.0, and I don't know what's in the latter. (I could find out, but for this PR I want to improve coverage of the main/apm_8.0.0 archive so I can upgrade it in a second PR, in preparation for the Metric Powered KPIs tests). Would you be okay with me opening up an issue around archive naming? I can pick it up after I merge Metric Powered KPIs.

Reply:
Sure feel free to merge. I don't think we need an issue for this. But having two almost identically named archives is going to cause confusion so we should fix it.